compiler/rustc_target/src/spec/targets/riscv64_wrs_vxworks.rs

```diff
@@ -0,0 +1,24 @@
+use crate::spec::{base, StackProbeType, Target, TargetOptions};
+
+pub(crate) fn target() -> Target {
+    Target {
+        llvm_target: "riscv64".into(),
+        metadata: crate::spec::TargetMetadata {
+            description: None,
+            tier: Some(3),
+            host_tools: Some(false),
+            std: None, // ?
```

Comment:
What is `// ?` for?

Reply:
`?` denotes std is a work in progress.

Reply:
Using words would be better :)  Here and for the 32-bit version above.